ibis/expr/sequel.py

```diff
@@ -0,0 +1,225 @@
+from __future__ import annotations
```

Comment:
Any issue with calling this `sql.py`? Seems a bit clever to call it `sequel.py` 😉